src/App.js

```diff
@@ -94,6 +94,7 @@ class App extends Component {
             <PublicRouteWithLegacyFallback exact path="/forgot-password" />
             <RedirectWithQuery exact from="/ingresa" to="/login?lang=es" />
             <RedirectWithQuery exact from="/registrate" to="/signup?lang=es" />
+            <RedirectWithQuery exact from="/olvidaste-contrasenia" to="/forgot-password?lang=es" />
```

Comment:
Ugh!

Please, review it with @carodipietro, "contrasenia" is very ugly!

Reply:
I will do so :)

Reply:
Is not *that* ugly anyways.

Reply:
@carodipietro what do you think about this?